core/src/main/kotlin/org/evomaster/core/search/gene/MapGene.kt

```diff
@@ -186,15 +200,57 @@ class MapGene<T>(
         elements.clear()
     }
 
-    fun removeElements(element: T){
+    /**
+     * remove [element] to [elements]
+     */
+    fun removeElements(element: PairGene<K, V>){
         elements.remove(element)
```

Comment:
how does this work? doe genes implements `equalsTo`?  or is this assuming heap reference? add clarification

Reply:
to clarify, how/where is this method supposed to be used?
if it is just a matter of removing an element, shouldn't just pass in a reference to the `K`?